packages/theme-samples/src/WordCustomizations.ts

```diff
@@ -1,42 +1,27 @@
 import { createTheme, ICustomizations } from 'office-ui-fabric-react';
 import { addVariants } from '@uifabric/variants';
+import { Theme } from '@fluentui/theme';
+
+export const WordTheme: Theme = createTheme({
+  palette: {
+    themePrimary: '#2b579a',
+    themeSecondary: '#366ec2',
+  },
+  semanticColors: {
+    buttonBackground: 'white',
```

Comment:
minor: what do you think about writing colors in **hex** format? Like in previous file.